app/src/main/java/org/oppia/app/player/audio/AudioFragment.kt

```diff
@@ -10,17 +10,38 @@ import org.oppia.app.model.State
 import javax.inject.Inject
 
 /** Fragment that controls audio for a content-card. */
-class AudioFragment : InjectableFragment(), LanguageInterface, AudioUiManager, CellularDataInterface {
+class AudioFragment : InjectableFragment(), LanguageInterface, AudioUiManager,
+  CellularDataInterface {
   @Inject lateinit var audioFragmentPresenter: AudioFragmentPresenter
 
+  companion object {
+    /**
+     * Creates a new instance of a AudioFragment.
+     * @param profileId used by AudioFragment to get Audio Language.
+     * @return a new instance of [AudioFragment].
+     */
+    fun newInstance(profileId: Int): AudioFragment {
+      val audioFragment = AudioFragment()
+      val args = Bundle()
+      args.putInt(AUDIO_FRAGMENT_PROFILE_ID_ARGUMENT_KEY, profileId)
+      audioFragment.arguments = args
+      return audioFragment
+    }
+  }
+
   override fun onAttach(context: Context) {
     super.onAttach(context)
     fragmentComponent.inject(this)
   }
 
-  override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+  override fun onCreateView(
+    inflater: LayoutInflater,
+    container: ViewGroup?,
+    savedInstanceState: Bundle?
+  ): View? {
     super.onCreateView(inflater, container, savedInstanceState)
-    return audioFragmentPresenter.handleCreateView(inflater, container)
+    val internalProfileId = arguments!!.getInt(AUDIO_FRAGMENT_PROFILE_ID_ARGUMENT_KEY, -1)
```

Comment:
Add `default value` here also.
`val internalProfileId = arguments!!.getInt(AUDIO_FRAGMENT_PROFILE_ID_ARGUMENT_KEY, /* defaultValue= */ -1)`
Also, here the default value is `-1` and the `AudioFragmentTestActivity` has default value is `0`. Make it consistent ideally it should be `-1` at both places.

Reply:
Done

Reply:
The inline comment needs to be added as per first comment:
`val internalProfileId = arguments!!.getInt(AUDIO_FRAGMENT_PROFILE_ID_ARGUMENT_KEY, /* defaultValue= */ -1)`

Reply:
> @nikitamarysolomanpvt There are 2 final nits which were not solved from last time, else everything is good. Also take latest pull from develop to pass circle CI

Done.

Reply:
> The inline comment needs to be added as per first comment:
> `val internalProfileId = arguments!!.getInt(AUDIO_FRAGMENT_PROFILE_ID_ARGUMENT_KEY, /* defaultValue= */ -1)`

Done